configuration/config-txt/video.md

```diff
@@ -469,6 +470,16 @@ Use `display_rotate` to rotate or flip the screen orientation. The default value
 
 Note that the 90 and 270 degree rotation options require additional memory on the GPU, so these will not work with the 16MB GPU split.
 
+### display_lcd_rotate
+
+Use `display_lcd_rotate` to rotate or flip the LCD orientation. Parameters are the same as `display_hdmi_rotate`.
+
+### display_rotate
+
+`display_rotate` is deprecated in the latest firmware but has been retained for backwards compatibility. Please use `display_lcd_rotate` and `display_hdmi_rotate` instead.
```

Comment:
I wonder if it'd be useful to document the date (or firmware-version) in which this change happened?

Reply:
Also, does the `display_hdmi_rotate` also apply to the composite video output?